keyboards/dztech/dz60rgb_ansi/v1/rules.mk

```diff
@@ -2,15 +2,13 @@
 MCU = STM32F303
 
 BACKLIGHT_ENABLE = no
-BOOTMAGIC_ENABLE = lite           # Virtual DIP switch configuration
+BOOTMAGIC_ENABLE = lite            # Virtual DIP switch configuration
 MOUSEKEY_ENABLE = yes              # Mouse keys
 EXTRAKEY_ENABLE = yes              # Audio control and System control
 CONSOLE_ENABLE = no                # Console for debug
 COMMAND_ENABLE = no                # Commands for debug and configuration
 #SLEEP_LED_ENABLE = yes            # Breathing sleep LED during USB suspend
-NKRO_ENABLE = no                 # USB Nkey Rollover
+NKRO_ENABLE = yes                  # USB Nkey Rollover
 AUDIO_ENABLE = no
 RGB_MATRIX_ENABLE = IS31FL3733     # Use RGB matrix
 NO_USB_STARTUP_CHECK = no          # Disable initialization only when usb is plugged in
-
-LAYOUTS = 60_ansi
```

Comment:
Is there a particular reason for this removal?

Reply:
If this is specifically due to community keymaps erroring out, that's okay.  Not all of them are tested on every keyboard. But as long as the default keymap compiles, then it should be fine.